seeders/20200427060316-initial.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('todos', [{
```

Comment:
ERROR: relation "todos" does not exist, coba di cek lagi untuk running seed nya

Reply:
iya bang tadi pas awal seed tabel name nya awalnya g paskal ku ganti manual di pgAdmin blom dirubah lagi di file seedersnya

Reply:
oke di perbaiki lagi ya kalo mau pake seed